_inc/client/components/settings-card/index.jsx

```diff
@@ -321,9 +321,20 @@ export const SettingsCard = props => {
 		return null;
 	}
 
+	let moduleClassName = '';
```

Comment:
@eliorivero Can we modify the variable name? As it is now an Id can it be moduleID or similar so that future devs can follow usage easier?

Thanks

Reply:
Good catch! Will update, thanks!